article-analyzer/src/main/java/co/inventorsoft/academy/spring/service/SlackNotificationService.java

```diff
@@ -0,0 +1,21 @@
+package co.inventorsoft.academy.spring.service;
+
+import co.inventorsoft.academy.spring.model.NotificationType;
+import co.inventorsoft.academy.spring.model.User;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Data
+@NoArgsConstructor
+@Service
+public class SlackNotificationService extends NotificationService {
+
+    private NotificationType type = NotificationType.SLACK;
```

Comment:
is type fields used?

Reply:
type field is used in NotificationServiceResolver class (directory : resolver). Resolver look at NotificationType fild in User class and compares with this field. So, I can get the specific instance of NotificationService.